app/assets/javascripts/controllers/ems_common/aggregate_status_card_controller.js

```diff
@@ -17,6 +17,7 @@ angular.module( 'patternfly.card' ).controller('aggregateStatusCardController',
         vm.showTopBorder = aggStatusData.showTopBorder;
         vm.aggregateLayout = aggStatusData.aggregateLayout;
         vm.aggregateClass = aggStatusData.aggregateClass;
+        return Promise.resolve();
```

Comment:
this will have no effect,
you're already in a promise, so this is identical to `return;`

Reply:
Removing that ugly leftover :)